api/alerts.go

```diff
@@ -70,6 +83,14 @@ func (a *Alerts) Update(viewName string, newAlert *Alert) (*Alert, error) {
 		field := graphql.String(newAlert.ThrottleField)
 		throttleField = &field
 	}
+	var runAsUserIdPtr *graphql.String
+	if newAlert.RunAsUserID != "" {
+		runAsUserIdPtr = graphql.NewString(graphql.String(newAlert.RunAsUserID))
+	}
+	var queryOwnershipType = humiographql.QueryOwnershipTypeUser
```

Comment:
I do not think that this should have a default value. Isn't the input value mandatory? In that case, I think that this should fail, if it does not match either of the two allowed values.

Reply:
The input is not mandatory and is getting defaulted to User.

![image](https://github.com/humio/cli/assets/158591046/9e753610-b875-4f89-afde-8a6382fc1b34)

So we could skip emitting it in the request (which is a bit cumbersome with a scalar and this lib), however it should result in the same object, and appears to do that when testing.

Reply:
This have been resolved by making an input struct with json omitempty on QueryOwnershipType which will omit on ""